src/pvnet_app/app.py

```diff
@@ -126,61 +121,78 @@ def get_boolean_env_var(env_var: str, default: bool) -> bool:
 
 
 def app(
-    t0=None | pd.Timestamp,
+    t0: None | pd.Timestamp = None,
     gsp_ids: list[int] = all_gsp_ids,
     write_predictions: bool = True,
     num_workers: int = -1,
 ):
-    """Inference function for production
+    """Inference function to run PVNet.
+
+    Args:
+        t0 (datetime): Datetime at which forecast is made
+        gsp_ids (array_like): List of gsp_ids to make predictions for. This list of GSPs are summed
+            to national.
+        write_predictions (bool): Whether to write prediction to the database. Else returns as
+            DataArray for local testing.
+        num_workers (int): Number of workers to use to load batches of data. When set to default
+            value of -1, it will use one less than the number of CPU cores workers.
 
     This app expects these environmental variables to be available:
         - DB_URL
         - NWP_UKV_ZARR_PATH
         - NWP_ECMWF_ZARR_PATH
         - SATELLITE_ZARR_PATH
     The following are options
-        - PVNET_V2_VERSION, pvnet version, default is a version above
-        - PVNET_V2_SUMMATION_VERSION, the pvnet version, default is above
```

Comment:
is possible could you update the readme on this project for the env vars. You might want to strip out the `environmental variables` text here as its then duplicate. Up to you